test/verification/stratified_couette_flow.jl

```diff
@@ -0,0 +1,264 @@
+using Statistics, Printf
+
+using Oceananigans
+using Oceananigans.TurbulenceClosures
+
+""" Friction velocity squared. See equation (16) of Vreugdenhil & Taylor (2018). """
+function uτ²(model)
+    Δz = model.grid.Δz
+    ν = model.closure.ν
+
+    Up = HorizontalAverage(model, model.velocities.u; frequency=Inf)
+    U = Up(model)[2:end-1]
+
+    uτ²⁺ = ν * abs(U[1] - Uw) / Δz
```

Comment:
Isn't this wrong? The distance between the `u` points and the wall is `Δz/2`, right?